umap/static/umap/js/umap.forms.js

```diff
@@ -939,6 +939,10 @@ L.U.FormBuilder = L.FormBuilder.extend({
 
   defaultOptions: {
     name: { label: L._('name') },
+    owner: {
+      label: 'owner',
+//      handler: 'BlurInput',  // this field should be hidden, not sure if blur does it.
```

Comment:
I don't think the [BlurInput](https://github.com/yohanboniface/Leaflet.FormBuilder/blob/master/Leaflet.FormBuilder.js#L290-L307) would achieve this. You probably need to extend `Input` with a [custom `type`](https://github.com/yohanboniface/Leaflet.FormBuilder/blob/master/Leaflet.FormBuilder.js#L271-L273) which returns `'hidden'`.